Assets/Scripts/Utilities/Settings.cs

```diff
@@ -149,15 +152,15 @@ private static void saveSettings()
 
     private static void loadSettings()
     {
-        // initilize the settings dict
+        // Initilize the settings dict.
```

Comment:
*&^#ing spelling mistake. Of course I catch it immediately _after_ doing the push. :stuck_out_tongue_closed_eyes: